build-a-react-native-application-for-ios-and-android-from-start-to-finish/notes/03-display-and-format-text-in-a-react-native-application.md

```diff
@@ -0,0 +1,53 @@
+# Display and Format Text in a React Native Application
+
+**[📹 Video](https://egghead.io/lessons/react-native-display-and-format-text-in-a-react-native-application)**
+
+## Components are like HTML elements
+
+HTML elements are not available in React Native, instead apps are composed of components imported from the `react-native` package. Most HTML elements have a comparable React Native component, the most basic being a `View` component - similar to a `<div>` - and a `Text` component - similar to a `<span>`.
+
+```jsx
+import { Text, View } from 'react-native'
+```
+
+The main difference between these components and their HTML equivalents is that text may only appear in a `<Text />` component, not a `<View />`. `<Text />` components may also contain other nested `<Text />` components but cannot contain nested `<View />` components. `<View />` components may contain other nested `<View />` or `<Text />` components.
+
+✅
+```jsx
+<View>
+  <Text>hello</Text>
+</View>
+```
+
+❌
+```jsx
+<View>hello</View>
+```
+
+| Component  | Contain `<View />` | Contain `<Text />` | Contain text |
```

Comment:
I like the markdown chart!

Reply:
Thanks! 🙂 I felt it a nice way to communicate the different options 👍